.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,71 @@
+# Modified for MoinMoin, copied from: https://github.com/actions/starter-workflows/blob/main/ci/python-package.yml
+# and https://github.com/borgbackup/borg/blob/master/.github/workflows/ci.yml
```

Comment:
maybe also rename this file to ci.yml, github-actions.yml is too generic.

also, non-ci stuff would be better put into a separate file.